src/mutation_data.cpp

```diff
@@ -132,7 +132,7 @@ void mutation_branch::load( JsonObject &jsobj )
         int neutral = jo.get_int("neutral", 0);
         int good = jo.get_int("good", 0);
         tripoint protect = tripoint(ignored, neutral, good);
-        new_mut.protection[part_id] = mutation_wet(body_parts[part_id], protect);
+        new_mut.protection[part_id] = mutation_wet(get_body_part_token( part_id ), protect);
```

Comment:
Well, does the mutation-wetness effect actually work? Because the map look-up would create a default constructed body_part (I assume it's `bp_torso = 0`), so all mutation based protection would have only been applied to the torso.

I'm in the process of fixing this. I apologize for not testing this, who could have guessed that this would be problem?